### ISSUE TITLE
Multiple volumes

### ISSUE THREAD

**Issue by yemi** [bitnami/bitnami-docker-wordpress#181]
I've been debugging this for quite some time now including browsing the issues.

Maybe I'm just getting tired but **this is what I want to do:**

1. Mount wp_uploads:/bitnami/wordpress/wp-content/uploads
2. Mount ./my-app/themes/my-theme:/bitnami/wordpress/wp-content/themes/my-theme

However, by doing (1) the folder will be owned by root and I'm thus not able to use it unless i manually change the permissions to e.g. 777. What would be the preferred way of achieving this since I wouldn't want to mount a folder on my own file system, right?
 
About (2), same issue here, folder gets owned by root and wordpress never finishes initialization because of this. Any ideas of how I can achieve these simple tasks?

**Comment by alemorcuq** [comment]
Hi, @yemi.

How are you trying to deploy Wordpress, can you share it? I've tried to reproduce your issue with `docker-compose` but I was able to successfully initiliaze Wordpress, and no folder is owned by root. Here's what I tried:

```docker-compose
version: '2'
services:
  mariadb:
    image: 'bitnami/mariadb:10.3'
    volumes:
      - 'mariadb_data:/bitnami'
    environment:
      - MARIADB_USER=bn_wordpress
      - MARIADB_DATABASE=bitnami_wordpress
      - ALLOW_EMPTY_PASSWORD=yes
  wordpress:
    image: 'bitnami/wordpress:5'
    ports:
      - '80:80'
      - '443:443'
    volumes:
      - 'wordpress_data:/bitnami'
      - './wp_uploads:/bitnami/wordpress/wp-content/uploads'
      - './my-theme:/bitnami/wordpress/wp-content/themes/my-theme'
    depends_on:
      - mariadb
    environment:
      - MARIADB_HOST=mariadb
      - MARIADB_PORT_NUMBER=3306
      - WORDPRESS_DATABASE_USER=bn_wordpress
      - WORDPRESS_DATABASE_NAME=bitnami_wordpress
      - ALLOW_EMPTY_PASSWORD=yes
volumes:
  mariadb_data:
    driver: local
  wordpress_data:
    driver: local
```

Regards,
Alejandro

**Comment by yemi** [comment]
Seems like if I first mount `wordpress_data:/bitnami`, stop the container, then mount `./my-theme`, and then restart the container, everything get the correct permissions.

If I, however, with no existing volumes, mount both of them, then `/bitnami/wordpress/wp-content` gets owned by root and wordpress cannot initialize.

This is my docker-compose.yml:

```
version: '3.7'

services:
  wordpress_db:
    image: 'bitnami/mariadb:10.3'
    volumes:
      - 'wordpress_db_data:/bitnami'
    environment:
      - MARIADB_USER=bn_wordpress
      - MARIADB_DATABASE=bitnami_wordpress
      - ALLOW_EMPTY_PASSWORD=yes
    
  wordpress:
    image: 'bitnami/wordpress:5'
    volumes:
      - './my-theme:/bitnami/wordpress/wp-content/themes/my-theme'
      - 'wordpress_data:/bitnami'
    depends_on:
      - wordpress_db
    environment:
      - MARIADB_HOST=wordpress_db
      - MARIADB_PORT_NUMBER=3306
      - WORDPRESS_DATABASE_USER=bn_wordpress
      - WORDPRESS_DATABASE_NAME=bitnami_wordpress
      - ALLOW_EMPTY_PASSWORD=yes
```

There is an nginx reverse proxy in front of these two containers.

To summarize:
1. If I run the containers with `./my-theme` mounted from the start, `/bitnami/wordpress/wp-content` and all its decendants get owned by root and thus wordpress wont initialize
2. If I first run the contaienrs without `./my-theme` to populate `wordpress_data`, then restart the containers with `./my-theme` mounted, it now works properly.

**Comment by alemorcuq** [comment]
Hi, @yemi.

I'm trying with your `docker-compose.yml` file, but it works well for me. Could you please me give me some step by step process to reproduce it?

You also mention Wordpress not being able to finish initialization, could you paste the logs too?

Regards,
Alejandro

**Comment by yemi** [comment]
I found the issue! It wasn't stuck at initialization, it was just taking a really long time, I left a node_modules in my theme that apparently was the cause of the long running initialization.

I read somewhere in another issue that the chown command could take a long time, is that what happened here as well? Why would that be?

**Comment by yemi** [comment]
Also @alemorcuq , thanks for the good support!

**Comment by alemorcuq** [comment]
Hi, @yemi.

I'm glad you were able to identify and solve your issue. You can set the environment variable `NAMI_LOG_LEVEL=trace8` in the Wordpress container to see what's happening, that way we can check if it was taking too long because of the chmod/chown operations.

Regards,
Alejandro
